scalecube-configuration/src/main/java/io/scalecube/configuration/ConfigurationServiceImpl.java

```diff
@@ -68,6 +70,11 @@ private ConfigurationServiceImpl(
         .doOnError(th -> logger.error("entries: request: {}, error: {}", request, th));
   }
 
+  @Override
+  public Mono<List<FetchResponse>> collectEntries(FetchRequest request) {
+    return entries(request).collectList();
```

Comment:
there should be some limit to the size of the list?
like take max 100?

Reply:
What is the business case to get a limited count of entries?

It could be useful only if you want to implement pagination logic, but right now we don't have such requirements.